### DISCUSSION TITLE
RDF

### DISCUSSION THREAD

**Discussion by RichDijk** [modular/modular#2357] [Q&A]
Hi,

Mojo looks promising as a Python superset. Maybe difficult to answer but, the RDFlib ecosystem is already large, how could it (semantic web stack) benefit from Mojo? Do e.g. plans exist for creating a RDF graphstore like Oxigraph in Rust?

Regards, Richard

**Comment by Benny-Nottonson** [comment]
I don't believe anyone has talked about plans for that kind of a library yet, and it likely wouldn't ever be supported directly by Modular, but you are welcome to create a project for it 👍
